update(platform): BYOC AWS GA (#3)
aiven/aiven-docs: docs/platform/howto/byoc/create-custom-cloud.md

    Console](https://console.aiven.io/).
    :::

1.  In your AWS account, run the template with the variables using
    Terraform.

    :::important
    When running `terraform plan` and `terraform apply`, make sure you
    add `-var-file=FILE_NAME.vars` as an option.
    :::

1.  Find the role identifier (Role ARN) in the output script after
    running the template.

1.  Enter Role ARN into the **Role ARN** field in the **Create custom
    cloud** wizard.

1.  Select **Next** to proceed or park your cloud setup and save
    your current configuration as a draft by selecting **Save draft**.
    You can resume creating your cloud later.

#### Set up your custom cloud's availability

Select in what projects you'll be able to use your new custom cloud as a hosting cloud for
services. In the projects where you enable your custom cloud, you'll be able to create new
services in the custom cloud and migrate your existing services to the custom cloud.

> You can make your cloud available as follows:
> 
> - For all the projects in your organization
> - For selected organizational units
> - For specific projects only

Reviewer: This would imply we are going into a list of steps make a cloud available, but this is not the case, so perhaps something like this:

```suggestion
You cloud can be available in:

- All the projects in your organization
- Selected organizational units
- Specific projects only
```

    list of available clouds and check its status in the **Status**
    column.

When your custom cloud's status is **Active**, its deployment has been completed. Your
custom cloud is ready to use and you can see it on the list of your custom clouds in the
**Bring your own cloud** view. Now you can create new services in the custom cloud and
migrate your existing services to the custom cloud.

#### Destroy the Terraform resources

As soon as you new custom cloud gets the **Active** status, remove the
Terraform resources your created in your AWS account while creating the
cloud. See the guidelines on how to use the `destroy` command in
[Command: destroy](https://developer.hashicorp.com/terraform/cli/commands/destroy).

## Manage services in custom clouds

### Create new services in the custom cloud

To create a service in [Aiven Console](https://console.aiven.io/) in your new custom
cloud, follow the guidelines in
[Create a new service](/docs/platform/howto/create_new_service).

When creating a service in the [Aiven Console](https://console.aiven.io/), at the
**Select service region** step, select **Custom clouds** from the available regions.

> ### Migrate existing services to the custom cloud

Reviewer 2: SEO is great! I hadn't thought of that. I'll review the draft

1.  Log in to [Aiven Console](https://console.aiven.io/) as an
    administrator.
1.  Select the organization you want to use from the dropdown menu in
    the top right corner.
1.  From the top navigation bar, select **Admin**.
1.  From the left sidebar, select **Bring your own cloud**.
1.  In the **Bring your own cloud** view, identify your new cloud on the
    list of available clouds and check its status in the **Status**
    column.

> If your custom cloud's status is **Active**, its deployment has been completed. Your

Reviewer: ```suggestion
When your custom cloud's status is **Active**, its deployment has been completed. Your
```
